crates/rspack_plugin_javascript/src/parser_and_generator/mod.rs

```diff
@@ -256,7 +256,7 @@ impl ParserAndGenerator for JavaScriptParserAndGenerator {
       SourceMapSource::new(SourceMapSourceOptions {
         value: output.code,
         name: resource_data.resource_path.to_string_lossy().to_string(),
-        source_map: SourceMap::from_json(&map).map_err(|e| internal_error!(e.to_string()))?,
```

Comment:
Always assume source maps passed back from swc is valid, otherwise, it's an internal bug